### PR TITLE
added vcd stat20-r-packages.r

### PR DESCRIPTION
**Pull request by shomurali**

### CODE REVIEW

File: stat20-r-packages.r [hunk 1]
```diff
@@ -23,7 +23,8 @@ set_libs <- c(
   "openintro", "2.4.0",
   "janitor", "2.2.0",
   "quarto", "1.2",
-  "fs", "1.6.3"
+  "fs", "1.6.3",
+  "vcd", "1.4-12"
```
Comment (andrewpbray) [inline stat20-r-packages.r:27]: should this be `1.4.12`?

Comment (shomurali) [reply stat20-r-packages.r:27]: Thats what I thought, but this is the page: https://cran.r-project.org/web/packages/vcd/index.html

Comment (shomurali) [reply stat20-r-packages.r:27]: And I looked at some other packages to see what they looked like and they are all of the form n.k1.k2

Comment (andrewpbray) [reply stat20-r-packages.r:27]: Hmm, well lets just test locally if the hyphen works. one sec.

Comment (andrewpbray) [reply stat20-r-packages.r:27]: Looks like it works with the hyphen!

```r
devtools::install_version("vcd", "1.4-12")
```

**Pull request merged**